### ISSUE TITLE
RPC: `hashAlgorithm` field missing in HTLC account responses

### ISSUE THREAD

**Issue by sisou** [nimiq/core-rs-albatross#2816]
The response object when requesting a HTLC account with the RPC method `getAccountByAddress()` does not include the hash algorithm type.

**Comment by sisou** [comment]
It's included as a sub-struct of `hashRoot`: 

```js
{
  address: 'NQ49 01C4 8C75 BD5K 18Q8 5Q08 DD49 H293 86LQ',
  balance: 1000,
  type: 'hTLC',
  sender: 'NQ58 VP2D VARE CYRQ 2QCB 8V1S CAM5 2QF1 39SA',
  recipient: 'NQ54 FTGY F6VJ EJPU NSMN RA5Q 0K21 8EQT Q05P',
  hashRoot: {
    algorithm: 'blake2b',
    hash: '0000000000000000000000000000000000000000000000000000000000000000'
  },
  hashCount: 1,
  timeout: 1702031666000,
  totalAmount: 1000
}
```

**Comment by sisou** [comment]
`type: 'hTLC'` camelCasing bug fixed in b2bf71424cf5579b23ded1580d2d506137f91346.
